apps/reference/_gotrue/usage.mdx

```diff
@@ -0,0 +1,402 @@
+---
+id: usage
+title: Usage
+description: An SWT based API for managing users and issuing SWT tokens
```

Comment:
I don't think we should write this one - it should be generated from the OpenAPI spec ideally - that way it will always stay in sync 

(we do this for storage etc)

Reply:
Agreed—this is just meant to be a short term fix. This change is really just moving [this page](https://supabase.com/docs/gotrue/server/about), which is currently "orphaned" (not included in any sidebar), to the Auth Server reference docs.